Dockerfile.assets

```diff
@@ -0,0 +1,16 @@
+ARG image_tag=latest
+FROM elifesciences/journal_composer:${image_tag} as composer
+FROM elifesciences/journal_npm:${image_tag} as npm
+FROM node:6.14.1
+
+COPY --from=npm /node_modules/ node_modules/
+
+COPY assets/images/ assets/images/
+COPY critical-css.json \
+    gulpfile.js \
+    ./
+COPY --from=composer /app/vendor/elife/patterns/resources/assets/ vendor/elife/patterns/resources/assets/
+
+RUN node_modules/.bin/gulp assets
```

Comment:
Is this equivalent to https://github.com/elifesciences/journal-formula/blob/master/salt/journal/scripts/retrying-gulp-without-redis.sh ?
There's definitely no Redis where the image is being built, and we can size memory appropriately. The multiple attempts may or may not be a solved problem.